doc/openstack/README.rst

```diff
@@ -0,0 +1,90 @@
+*************************
+Molecule Openstack Plugin
+*************************
+
+Molecule Openstack is designed to allow use of Openstack
+for provisioning of test resources.
+
+.. _quickstart:
+
+Quickstart
+==========
+
+Installation
+------------
+
+.. code-block:: bash
+
+    pip install molecule-plugins
+
+Create a scenario
+-----------------
+
+With a new role
+^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   molecule init role -d openstack my-role
```

Comment:
This command is not available now.

Reply:
I am not the author of the PR but I would like to help to get the OpenStack plugin merged in this repository. I would not mind to patch this PR and create a new one (not sure if there is a better way to do this).

Reply:
@juanlufont 

Thanks for your attention.

A new PR is fine for me. I'll review it.